packages/lesswrong/server/repos/PostsRepo.ts

```diff
@@ -1239,6 +1239,50 @@ class PostsRepo extends AbstractRepo<"Posts"> {
       };
     });
   }
+
+  /**
+   * Get posts from users the current user is subscribed to, for UltraFeed
+   */
+  async getPostsFromSubscribedUsersForUltraFeed(
+    context: ResolverContext,
```

Comment:
If the only thing you're using the context for is to pass in a userId, just pass that in directly and skip the null check later.